scripts/build.js

```diff
@@ -1079,6 +1213,19 @@ export const buildEngine = async (options) => {
   // Create index.js
   await createIndexJs("engine");
 
+  await bundleIndexJs({
```

Comment:
Reminder for myself. These need hooked up in the dev `server.js` to rebuild when sandcastle refreshes if there have been changes. Right now it only does the top level files.